From: =?utf-8?q?Florian_M=C3=BCllner?= <fmuellner@gnome.org>
Date: Tue, 5 Mar 2019 19:42:01 +0100
Subject: st-widget: Add missing g_return_val_if_fail()

It's a public function, so it's good practice to sanity-check its
input (unless disabled at compile time of course).

Bug: https://gitlab.gnome.org/GNOME/gnome-shell/issues/1018
Origin: upstream, 3.33.0, commit:706bdd805912fb7d2ea7c630465ae733be4bdd26
---
 src/st/st-widget.c | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/src/st/st-widget.c b/src/st/st-widget.c
index db984ac..236e03d 100644
--- a/src/st/st-widget.c
+++ b/src/st/st-widget.c
@@ -598,7 +598,11 @@ get_root_theme_node (ClutterStage *stage)
 StThemeNode *
 st_widget_get_theme_node (StWidget *widget)
 {
-  StWidgetPrivate *priv = st_widget_get_instance_private (widget);
+  StWidgetPrivate *priv;
+
+  g_return_val_if_fail (ST_IS_WIDGET (widget), NULL);
+
+  priv = st_widget_get_instance_private (widget);
 
   if (priv->theme_node == NULL)
     {
